src/Application/Application.csproj

```diff
@@ -12,12 +12,15 @@
             <PrivateAssets>all</PrivateAssets>
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
+        <PackageReference Include="Microsoft.SemanticKernel" Version="1.0.0-beta3" />
```

Comment:
Should Semantic Kernel be part of `Application`?  Is this more of an `Infrastructure` concern?